beat/models/seismic.py

```diff
@@ -936,7 +936,161 @@ def update_weights(
                 dataset.covariance.update_slog_pdet()
                 wmap.weights[i].set_value(choli)
 
+##Mahdi
+class SeismicPolarityComposite(Composite):
 
+    def __init__(self, polc, project_dir, sources, events, hypers=False):
+        super(SeismicPolarityComposite, self).__init__(events)
+        
+        self.name = 'polarity'
+        self._like_name = 'pol_like'
+        self.synthesizers = {}
+        self.sources = sources    
+        self.config = polc
+        self.sigma = 0.16
```

Comment:
> Where does that come from? From the paper? Please add a note as a comment!
> This would be the noise scaling (hyper) parameter? This would need to be handled properly as well. Similar to the other h_any_P parameter for example from seismic waveforms. Of course we need here a different nameing. maybe h_any_P_pol ... any_P of course would then again need to be taken from the PolarityMap definition ... Of course for now you can also keep it hardcoded here ..., Jan loves appropriate noise scaling/estimation ;)
I am going to take care of it.

Reply:
in get_formula():
hp = hyperparams[self.get_hypernames()]